sale_product_identification/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,4 @@
+This module allows you to add required IDs to products and validate those identifications when confirming a sales order.
```

Comment:
This should a little bit explained (which IDs ?).

A CONTEXT in readme should be great to explain it. Thanks